### DISCUSSION TITLE
how to change text color via for_window

### DISCUSSION THREAD

**Discussion by ericx** [i3/i3#5870] [Q&A] [answered]
I use a dark solaris scheme for my desktop; so the text color is white.
There are a number of apps still with white background (Thunderbird) and the text's color is black (don't know what mechanism is toggling that). However, when I run zscaler, it comes up with a white background and white text; so nothing is readable.

Is there a way to change the text color via `for_window [class="ZSTray"]`?

**Comment by orestisfl** [comment, marked as answer]
i3 doesn't control what windows display. You could however try controlling the theme of that specific application and see if you can achieve something: https://unix.stackexchange.com/questions/14129/gtk-enable-set-dark-theme-on-a-per-application-basis

**Comment by ericx** [comment]
Good answer; but the GTK_THEME variable is ignored as far as I can tell.
But not an i3 thing. 
Thank you.
